uber/site_sections/registration.py

```diff
@@ -17,16 +17,9 @@ def pre_checkin_check(attendee, group):
     if group and attendee.paid == c.PAID_BY_GROUP and group.amount_unpaid:
         return 'This attendee\'s group has an outstanding balance of ${}'.format(group.amount_unpaid)
 
-    if attendee.paid == c.PAID_BY_GROUP and not attendee.group_id:
-        return 'You must select a group for this attendee.'
-
     if attendee.paid == c.NOT_PAID:
         return 'You cannot check in an attendee that has not paid.'
 
-    attendee._status_adjustments()
-    if attendee.badge_status != c.COMPLETED_STATUS:
-        return 'This badge is {} and cannot be checked in.'.format(attendee.badge_status_label)
```

Comment:
We decided it was preferable to remove this check for two reasons:

1) We trip on it when claiming a group badge.

2) We don't even let people attempt to submit the form if the status isn't in a good state, so this isn't adding a lot of extra sanity anyway.